packages/angular-sdk-components/src/lib/_components/field/currency/currency.component.ts

```diff
@@ -7,7 +7,9 @@ import { interval } from 'rxjs';
 import { AngularPConnectData, AngularPConnectService } from '../../../_bridge/angular-pconnect';
 import { Utils } from '../../../_helpers/utils';
 import { ComponentMapperComponent } from '../../../_bridge/component-mapper/component-mapper.component';
+import { handleEvent } from '../../../_helpers/event-util';
 import { getCurrencyCharacters } from '../../../_helpers/currency-utils';
+import { NgxCurrencyDirective } from 'ngx-currency';
```

Comment:
Move this import above the local imports(in other components also).

Reply:
Done

Reply:
They still aren't above the local imports.